packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
   * Migrate docs from terra-layout
   * Added docs for `terra-filter-pills`
   * Added tests for `terra-filter-pills`
+  * Migrate docs from terra-tabs
```

Comment:
This needs to be moved to the Unreleased section.

Reply:
changed 4f8fb6f6d8f49d12eee55c6b6143e2e7d0cb80ae